### DISCUSSION TITLE
Icons not showing up in the top tab bar, but they are in the bottom tab bar. Why?

### DISCUSSION THREAD

**Discussion by mprogers** [dotnet/maui#16314] [Q&A]
I've got a shell-based .NET MAUI app. I have icons that are appearing in the bottom tab bar, but not in the top one.  

Here's the xaml:

`<?xml version="1.0" encoding="UTF-8" ?>
<Shell
    x:Class="MauiStockTake.UI.AppShell"
    xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
    xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
    xmlns:Pages="clr-namespace:MauiStockTake.UI.Pages"
    xmlns:local="clr-namespace:MauiStockTake.UI">
 
    <TabBar>
        <Tab Title="Input"
             Icon="icon_input.svg" >
            <ShellContent Title="Hello" Icon="icon_input.svg"  ContentTemplate="{DataTemplate Pages:InputPage}" />
            <ShellContent Title="Goodbye" Icon="icon_report.svg" ContentTemplate="{DataTemplate Pages:ReportPage}" />
        </Tab>
        <Tab Title="Reports"
             Icon="icon_report.svg">
            <ShellContent ContentTemplate="{DataTemplate Pages:ReportPage}" />
        </Tab>
    </TabBar>
 
</Shell>`

The icons show up in the bottom tab bar, but not at the top. Any idea why?

**Comment by JuanFcoMiranda** [comment]
Same problem here.

**Comment by Ashishkumar5699** [comment]
@jamesmontemagno There is still no update regarding the fixes of the top icon and badge.
